internal/docstore/dynamodocstore/dynamo.go

```diff
@@ -30,7 +30,8 @@
 //
 // dynamodocstore exposes the following types for As:
 //  - Collection.As: *dynamodb.DynamoDB
-//  - ActionList.BeforeDo: *dynamodb.BatchGetItemsInput or *dynamodb.TransactWriteItemsInput
+//  - ActionList.BeforeDo: *dynamodb.BatchGetItemInput, *dynamodb.PutItemInput, *dynamodb.DeleteItemInput,
```

Comment:
I think this should say "or" to make it more clear that each call will support one of these, not all.

Reply:
please fix

Reply:
Done.